baseplate/server/__init__.py

```diff
@@ -374,11 +374,9 @@ def load_and_run_shell() -> None:
 
     try:
         # try to use IPython if possible
-        from IPython.terminal.embed import InteractiveShellEmbed
-        from IPython.core.interactiveshell import DummyMod
-
-        shell = InteractiveShellEmbed(banner2=banner)
```

Comment:
It looks like the banner isn't getting shown.

Reply:
```
ubuntu@reddit:~/src/harold2 (harold2)$ onevm shell
Running “baseplate-shell /home/ubuntu/src/harold2/example.ini”…
INFO:raven.base.Client:Raven is not configured (logging is disabled). Please see the documentation for more information.
Python 3.8.5 (default, Jul 20 2020, 19:48:14) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: context
Out[1]: <baseplate.RequestContext at 0x7fdd587c3610>

```

so the values are there as expected but there's no message explaining how to use 'em.

Reply:
Good catch, let me get the banner back into place! (I'm so used to it at this point I've gloss right over it)